src/frontend/cxx03.y.in

```diff
@@ -4891,6 +4893,9 @@ nested_name_specifier_0 : class_or_namespace_name
     $$ = $1;
 }
 | decltype_specifier
+{
+	$$ = $1;
```

Comment:
This is a limitation of the current implementation of graph-glr and it will be lifted in the future. That said, the rest of the file uses this style so I think this is OK.